packages/lab/stories/form-field-next/form-field-next.doc.stories.mdx

```diff
@@ -0,0 +1,156 @@
+import { FormFieldNext } from "@salt-ds/lab";
+import { Meta, Canvas, ArgsTable, Story } from "@storybook/addon-docs";
+import {
+  CSSClassTable,
+  CharacteristicUsage,
+} from "css-variable-docgen-components";
+import HelpAndSupport from "docs/blocks/help-and-support.doc.mdx";
+
+<Meta
+  title="Documentation/Lab/Form Field Next"
+  component={FormFieldNext}
+  parameters={{
+    viewMode: "docs",
+  }}
+/>
+
+# Form Field Next
+
+Form Field is a wrapper for UI controls that are typically found in a form (for example, Input, Combobox or Radio Button). It is required to make these controls accessible by providing them with a visible label, validation control and states and descriptive text elements.
+
+<Canvas>
+  <Story id="lab-form-field-next--default" />
+</Canvas>
+
+# Examples
+
+All examples demonstrate Salt form controls inside the Form Field, including Input, Checkbox and Radio Button. These components are available as part of the Salt component library in addition to Form Field.
+
+## Form Controls
+
+Form field can wrap any Salt UI control that is typically found in a form, such as Input, Checkbox and Radio Button components.
+
+<Canvas>
+  <Story id="lab-form-field-next--with-controls" />
+</Canvas>
+
+### Guidance
+
+- Form Field doesn’t have primary and secondary variants. Any variants apply to the field controls contained within Form Field and can be displayed via the components `variant` prop if supported.
```

Comment:
```suggestion
- Form Field doesn’t have variants. Any variants apply to the field controls contained within Form Field and can be displayed via the components `variant` prop if supported.
```
Should this be "form controls contained withing Form Field" instead of "field controls contained within Form Field"

Reply:
I think mentioning any variants is unnecessary here